engine/src/flutter/shell/platform/windows/flutter_windows_view.cc

```diff
@@ -18,6 +19,11 @@
 namespace flutter {
 
 namespace {
+// The windows API sends pressure as a normalized value between 0 and 1024
+// See
+// https://learn.microsoft.com/en-ca/windows/win32/api/winuser/ns-winuser-pointer_pen_info
```

Comment:
The Canadian locale might not be the best one for the user, I'd let Microsoft's docs redirect :)

```suggestion
// https://learn.microsoft.com/windows/win32/api/winuser/ns-winuser-pointer_pen_info
```